### DISCUSSION TITLE
What an amazing piece of software!!

### DISCUSSION THREAD

**Discussion by kyr0** [JeffreyCA/spleeter-web#75] [Show and tell]
Hey :)

as a musician and web developer I know how much effort and skills it took to build this.
I was just researching the AI algos behind Music Rebalance and Unmix. It was hard to believe that those plugin developers did the models on their own. Next I stumbled upon the research paper, bumped into Spleeter (open source by Deezer) -- man I was happy already. Then I thought I would build a web GUI. But man... there is it already!!!! And it has more features than I thought of :)) 

Thank you!!

Best from Munich
Aron

**Comment by JeffreyCA** [comment]
Hi Aron, thank you for the kind words! I'm simply creating something I love and enjoy, and sharing it so others can experience the same 😃 

I agree, the real shoutout goes to all of the folks behind the research and the hard work they put in to make these technologies possible, and open-sourcing it of course!
